deps/wrapper.cc

```diff
@@ -182,33 +182,29 @@ ray::core::CoreWorker &_GetCoreWorker() {
     return CoreWorkerProcess::GetCoreWorker();
 }
 
-// https://github.com/ray-project/ray/blob/ray-2.5.1/src/ray/core_worker/test/core_worker_test.cc#L224-L237
-ObjectID put(std::shared_ptr<Buffer> buffer) {
+// Example of using `CoreWorker::Put`: https://github.com/ray-project/ray/blob/ray-2.5.1/src/ray/core_worker/test/core_worker_test.cc#L224-L237
+ObjectID put(const std::shared_ptr<RayObject> object,
+             const std::vector<ObjectID> &contained_object_ids) {
```

Comment:
aren't the contained object IDs a field in the ray object itself?  I guess the core worker Put function does take this separately: https://github.com/beacon-biosignals/ray/blob/4ceb62daaad05124713ff9d94ffbdad35ee19f86/src/ray/core_worker/core_worker.cc#L1110-L1112

Reply:
I also find this strange. I'll need to do some experimentation with this yet to determine why both are needed but for this PR in particular it seemed good to provide the option for experimentation.

Reply:
yup, no reason not to match the interface from the c++ function here